docs/source/diagrams/ReflectometryReductionOneAuto-v2-Groups_wkflw.dot

```diff
@@ -0,0 +1,82 @@
+digraph ReflectometryReductionOne {
+  label = "\n"
+  $global_style
+
+  subgraph params {
+    $param_style
+    floodWorkspace    [label="FloodWorkspace"]
+    inputWorkspace    [label="InputWorkspace"]
+    trans             [label="FirstTransmissionRun"]
+    IvsLam_0          [label="OutputWorkspaceWavelength"]
+    IvsLam_1          [label="OutputWorkspaceWavelength"]
+    IvsLam_temp       [label="OutputWorkspaceWavelength"]
+    IvsQ_0            [label="OutputWorkspace"]
+    IvsQ_1            [label="OutputWorkspace"]
+    IvsQ_temp         [label="OutputWorkspace"]
+    IvsLam_final      [label="OutputWorkspaceWavelength"]
+    IvsQ_2            [label="OutputWorkspace"]
+    IvsQ_3            [label="OutputWorkspace"]
+    IvsQ_final        [label="OutputWorkspace"]
+  }
+
+  subgraph algorithms {
+    $algorithm_style
+    reflRedOne_0 [label="ReflectometryReductionOne", group=g1]
+    reflRedOne_1 [label="ReflectometryReductionOne", group=g2]
+    groupIvsQ    [label="GroupWorkspaces", group=g1]
+    groupIvsLam  [label="GroupWorkspaces", group=g2]
+    polCorr      [label="PolarizationCorrection", group=g1]
+    reflRedOne_2 [label="ReflectometryReductionOne"]
+    reflRedOne_3 [label="ReflectometryReductionOne"]
+    groupFinal   [label="GroupWorkspaces", group=g1]
+    applyFloodI  [label="ApplyFloodWorkspace", group=g1]
+    applyFloodT  [label="ApplyFloodWorkspace", group=g1]
```

Comment:
Next to `ApplyFloodWorkspace`, there is also `CreateFloodWorkspace` in use and it would be good to mention it somewhere in the documentation. I could not find it except in the source code.